src/daniil_efremov_ex/pa02/test_pa02.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Daniil Efremov', 'Sigurd Grøtan'
+__email__ = 'daniil.vitalevich.efremov@nmbu.no', 'sgrotan@nmbu.no'
+
+import chutes_simulation as cs
+import pytest
+
+
+class TestBoard:
+    """
+    Tests for Board class.
+    """
+
+    def test_goal_reached(self):
+        """Test that checks whether 2 positions have reached goal or not"""
+
+        position_1 = 80
+        position_2 = 100
+        b = cs.Board(goal=90)
+        assert b.goal_reached(position_1) is False
+        assert b.goal_reached(position_2) is True
```

Comment:
Please have the habit of omitting `is True` when checking something is True, and using `not` when only checking for `False`